runners/spark/pom.xml

```diff
@@ -112,29 +112,11 @@
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
-      <artifactId>spark-streaming-kafka_2.10</artifactId>
```

Comment:
(Commenting here because related lines not part of the diff) Can you add a `-Plocal-runnable-on-service-tests-streaming` profile for easy launching?